### PR TITLE
Implement cache for html code of pages

### PR DESCRIPTION
**Pull request by tormozz48**
@tadatuta @tavriaforever @gela-d

### CODE REVIEW

**Comment by tavriaforever** [comment]
super fast! cool

**Comment by tormozz48** [comment]
Ho-ho guys! I almost forgot about pages localization. Please do not merge this pr. I will support it

File: src/blocks/server.blocks/middleware/__html-cache/middleware__html-cache.node.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+                logger.debug('load page from cache: %s', pagePath);
+
+                var raw = fs.createReadStream(pagePath);
+                var acceptEncoding = req.headers['accept-encoding'];
+                if (!acceptEncoding) {
+                    acceptEncoding = '';
+                }
+
+                if (acceptEncoding.match(/\bgzip\b/)) {
```
Comment (tadatuta) [inline src/blocks/server.blocks/middleware/__html-cache/middleware__html-cache.node.js:25]: my idea was to alway store static in gzip which would reduce HD usage and CPU load.
all browsers support it.

Comment (tormozz48) [inline src/blocks/server.blocks/middleware/__html-cache/middleware__html-cache.node.js:25]: I agree with you. So if you look it throught more deeper you will see that I archive cached files before save them.

Comment (tadatuta) [inline src/blocks/server.blocks/middleware/__html-cache/middleware__html-cache.node.js:25]: oh, I see!

File: src/blocks/server.blocks/middleware/__html-cache/middleware__html-cache.node.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+            fs.exists(pagePath, function (exists) {
+                if(!exists) {
+                    return next();
+                }
+
+                logger.debug('load page from cache: %s', pagePath);
+
+                var raw = fs.createReadStream(pagePath);
+                var acceptEncoding = req.headers['accept-encoding'];
```
Comment (tadatuta) [inline src/blocks/server.blocks/middleware/__html-cache/middleware__html-cache.node.js:20]:
``` js
acceptEncoding = req.headers['accept-encoding'] || '';
```

**Comment by tadatuta** [comment]
@tavriaforever was i18n implemented?

**Pull request merged**